tests/helpers.py

```diff
@@ -397,6 +397,31 @@ def reference_id_from_url(url):
     }
 ]
 
+TEST_SCENE_CATEGORY_ANNOTATIONS = [
+    {
+        "label": f"[Pytest] Scene Category Label ${i}",
+        "reference_id": reference_id_from_url(TEST_IMG_URLS[i]),
+        "taxonomy_name": "[Pytest] Scene Category Taxonomy 1",
```

Comment:
There's a dependency here with a taxonomy. If I would like to make more tests using this data structure I'd have to make sure this taxonomy exists. That's not a good constant. 

I'd make a fixture that adds the taxonomy name where it is used.